draft-ietf-gnap-core-protocol.md

```diff
@@ -4073,9 +4074,11 @@ containing the following information:
 
 
 grant_request_endpoint (string)
-: REQUIRED. The full URL of the
-          AS's grant request endpoint. This MUST match the URL the client instance used to
-          make the discovery request.
+: REQUIRED. The location of the
+          AS's grant request endpoint. The location MUST be a URI {{RFC3986}}
```

Comment:
Since this URI has a fixed scheme, it is more properly a URL. I'm not sure if the reference should remain this RFC or point to 3305 or something else?

```suggestion
          AS's grant request endpoint. The location MUST be a URL {{RFC3986}}
```

Reply:
I don't have any preference as to 3305 vs 3986.